tests/plugins/org.polarsys.capella.test.diagram.tools.ju/src/org/polarsys/capella/test/diagram/tools/ju/common/SelectToolsTest.java

```diff
@@ -81,16 +140,19 @@ public void test() throws Exception {
     Session s = getSession(getRequiredTestModel());
     SessionContext sc = new SessionContext(s);
 
-    testSelectRelatedFCElements(sc, PAB_DIAGRAM, FC1, FE1, FCAT1, P1, P2, P3, P4);
-    testSelectRelatedPPElements(sc, PAB_DIAGRAM, PPath1, PL1, PCAT1, PP1, PP2, PP3, PP4);
+    testCategoriesSemanticElements(sc);
+    //
+    // testSelectRelatedFCElements(sc, PAB_DIAGRAM, FC1, FE1, FE1_PORT1, FE1_PORT2, FCAT1_EDGE, FCAT1_PORT1,
```

Comment:
I hadn't seen. Is it normal? Can you uncomment or remove it?

Reply:
Not normal , Fixed it